components/Python/Scikit-learn/xgboost-train/xgboost_train.py

```diff
@@ -0,0 +1,320 @@
+import argparse
+import subprocess
+import sys
+
+import numpy as np
+import pandas as pd
+from scipy.stats import ks_2samp
+from sklearn.datasets import make_classification
+from sklearn.metrics import accuracy_score, roc_auc_score, roc_curve
+from sklearn.model_selection import train_test_split
+
+from parallelm.mlops import StatCategory as st
+from parallelm.mlops import mlops as mlops
+from parallelm.mlops.stats.bar_graph import BarGraph
+from parallelm.mlops.stats.graph import MultiGraph
+from parallelm.mlops.stats.table import Table
+
+def install(package):
+    subprocess.call([sys.executable, "-m", "pip", "install", package])
+
+
+def parse_args():
+    """
+    Parse arguments from component
+    :return: Parsed arguments
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--num_features", help="# Features")
```

Comment:
would it make sense to have the same defaults here also? so that one can run the program standalone?

Reply:
let it be. will come to it later if people wanna use standalone 🗡